src/alfasim_sdk/result_reader/aggregator.py

```diff
@@ -85,6 +92,120 @@ class ResultsNeedFullReloadError(RuntimeError):
     """
 
 
+@attr.s(slots=True, hash=False)
+class GlobalSensitivityAnalysisMetadata:
+    """
+    A class that hold the global sensitivity analysis metadata.
+    """
+
+    @attr.s(slots=True, hash=False)
+    class GSAItem:
```

Comment:
@prusse-martin please double check if this is what you meant by your previous comment. 👍 

https://github.com/ESSS/alfasim-sdk/pull/357#discussion_r1482897930

Reply:
I was more about a module level `GlobalSensitivityAnalysisMetadataItem` or a class scope (like it is here) `GlobalSensitivityAnalysisMetadata.Item`.

But `GlobalSensitivityAnalysisMetadata.GSAItem` is OK.